tools/packaging/static-build/coco-guest-components/build.sh

```diff
@@ -39,17 +39,17 @@ docker pull ${container_image} || \
 	 push_to_registry "${container_image}")
 
 # Temp settings until we have a matching TEE_PLATFORM
-TEE_PLATFORM=""
-RESOURCE_PROVIDER="kbs,sev"
+TEE_PLATFORM="${TEE_PLATFORM:-all}"
```

Comment:
Hi @huoqifeng - when I originally wrote this I had to explicit override the `TEE_PLATFORM` to none (and instead specify the `KBC_PROVIDER` and `ATTESTER` fields) as `all` forced the tss to be build into for the vTPM attesters and that wasn't required, or working. I was told there were plans to create a separate `TEE_PLATFORM` that didn't bring on there dependencies, but I don't think that has been done (or is under `ALL`).

Have things changes in guest-components where we expect this to work now?

Reply:
@stevenhorsman I want bring the option `TEE_PLATFORM=se` when build guest components.

Reply:
Ok, could you achieve this with setting `${TEE_PLATFORM:+all}` (so if it isn't set it won't be overridden. Similarly I assume you want `RESOURCE_PROVIDER="${RESOURCE_PROVIDER:+kbs,sev}"`, so that if you set `TEE_PLATFORM=se` then this isn't run?

The alternative without using `TEE_PLATFORM` is to set `RESOURCE_PROVIDER=se` for secure execution?

Reply:
Hmm, it looks like that operator doesn't work in the way I was expecting. Back to the drawing board I guess...

Reply:
Oh, I can see `se-attester` is not included in `all-attesters`: https://github.com/confidential-containers/guest-components/blob/dd2984d6c8bbb30205bbbff67cbb4c95c516224f/attestation-agent/attestation-agent/Cargo.toml#L61 So I am fine with `TEE_PLATFORM=${TEE_PLATFORM:-all}`. 😉 

@huoqifeng I think https://github.com/confidential-containers/guest-components/blob/dd2984d6c8bbb30205bbbff67cbb4c95c516224f/README.md?plain=1#L38 should be updated to reflect the current Cargo.toml for AA.

Reply:
Yes, let me open a PR to do this -- there is a problem to build some other attesters on s390x although it should ne OK to build IBM SE attester on x86. I think build all attester on x86 is our case.

Reply:
https://github.com/confidential-containers/guest-components/pull/590 created